modules/configuring-hybrid-ovnkubernetes.adoc

```diff
@@ -75,11 +75,13 @@ spec: <1>
         hybridClusterNetwork: <4>
         - cidr: 10.132.0.0/14
           hostPrefix: 23
+        hybridOverlayVXLANPort: 9898 <5>
 status: {}
 ----
 <1> The parameters for the `spec` parameter are only an example. Specify your configuration for the Cluster Network Operator in the custom resource.
 <2> Specify the CIDR configuration used when adding nodes.
 <3> Specify `OVNKubernetes` as the Container Network Interface (CNI) cluster network provider.
 <4> Specify the CIDR configuration used for nodes on the additional overlay network. The `hybridClusterNetwork` CIDR cannot overlap with the `clusterNetwork` CIDR.
+<5> Specify a custom VXLAN port for the additional overlay network. This is required for running Windows nodes in a cluster installed on vSphere; the custom port can be any open port excluding the default `4789` port.
```

Comment:
Should we link to [why](https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/common-problems#pod-to-pod-connectivity-between-hosts-is-broken-on-my-kubernetes-cluster-running-on-vsphere)?

Reply:
Sure; I'll add the link for some background.